parley/Cargo.toml

```
@@ -31,3 +31,6 @@ fontique = { workspace = true }
 core_maths = { version = "0.1.0", optional = true }
 accesskit = { workspace = true, optional = true }
 hashbrown = { workspace = true, optional = true }
+
+[dev-dependencies]
+tiny-skia = "0.11"
```

Let's use the full version.

```suggestion
tiny-skia = "0.11.4"
```